sdk/core/core-http-client-xhr/src/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should we also expose one for `react-native` as XHR is supported there such as `index-react-native.mts`?